erpnext_documentation/www/docs/v13/user/manual/en/erpnext_integration/shopify_integration_app.md

```diff
@@ -0,0 +1,120 @@
+<!-- add-breadcrumbs -->
+# Shopify Integration - Ecommerce Integrations app
+
+The Shopify Connector pulls the orders from Shopify and creates Sales Order against them in ERPNext.
+
+While creating the sales order if Customer or Item is missing in ERPNext the system will create new Customer/Item by pulling respective details from Shopify.
+
+### How to Setup Connector?
+
+
+#### Note to users of old Shopify Connector
+
+If you have not setup Shopify Connector on your ERPNext site you can proceed to next step.
+
+If you are using old Shopify integration that's provided in ERPNext then you will have to disable the connector before proceeding. After installing the app, it will migrate existing data e.g. unique product_id for items to separate doctype. Once you are done configuring new integration, you can confirm the status of migration by going to "Ecommerce Integration Log" doctype.
+
+#### App Installation
+
+- If you are hosting your ERPNext site on Frappe Cloud, you can quickly install the app by going to your site Dashboard. The app is available in [Frappe Cloud Marketplace](https://frappecloud.com/marketplace/apps/ecommerce-integrations)
+- If you want to install it on Private bench on Frappe Cloud refer [Frappe Cloud documentation](https://frappecloud.com/docs/bench/install-custom-app)
+- If you are self hosting ERPNext you can install the app using Frappe bench. Refer [bench documentation](https://frappeframework.com/docs/user/en/bench/frappe-commands#app-installation) for installing Frappe Apps.
+
+The repository for app is hosted on GitHub: [http://github.com/frappe/ecommerce_integrations/](http://github.com/frappe/ecommerce_integrations/)
+
+#### Create A Private App in Shopify
+
+1. Click on Apps in menu bar
+<img class="screenshot" alt="Menu Section" src="{{docs_base_url}}/v13/assets/img/erpnext_integrations/app_menu.png">
+
+2. Click on **Manage private apps** to create private app
+<img class="screenshot" alt="Manage Private Apps" src="{{docs_base_url}}/v13/assets/img/erpnext_integrations/manage_private_apps.png">
+
+3. Fill up the details and create app. The each app has its own API key, Password and Shared secret
+<img class="screenshot" alt="App Details" src="{{docs_base_url}}/v13/assets/img/erpnext_integrations/app_details.png">
+
+
+#### Setting Up Shopify  on ERPNext:-
+Once you have created a Private App on Shopify, setup App Credentials and other details in Shopify Setting in ERPNext.
+
+> To access Shopify Setting, go to:
+Awesome bar > "Shopify Setting"
```

Comment:
Instead of 'Awesome Bar', please provide the navigation path from desk...

Reply:
Right now there isn't any. I'll add it in docs, when I've updated the app.

Reply:
Moving ahead with merge..  will update this once it's added in app.